frontend/components/Header/Nav/Pillars/index.tsx

```diff
@@ -103,21 +102,50 @@ const linkStyle = css`
         transform: translateY(-4px);
     }
 `;
+const pillarUnderline = pillarMap(
+    pillar => css`
+        :after {
+            border-bottom: 4px solid ${pillarPalette[pillar].dark};
+        }
+    `,
+);
+
+const forceUnderline = css`
+    :after {
+        transform: translateY(-4px);
+    }
+    :focus:after {
+        transform: translateY(-4px);
+    }
+    :hover {
+        text-decoration: none;
+    }
+    :hover:after {
+        transform: translateY(-4px);
+    }
+`; // A11Y warning: this styling has no focus state for the selected pillar
```

Comment:
Is it easy to get the focus state back? I'd rather not make the article even less accessible for the A/B test